eval/run.rkt

```diff
@@ -99,7 +97,8 @@
 
 (define (rival-machine-adjust machine)
   (define iter (rival-machine-iteration machine))
-  (unless (zero? iter)
-    (define start (current-inexact-milliseconds))
-    (backward-pass machine)
+  (let ([start (current-inexact-milliseconds)])
+    (if (zero? iter)
+        (vector-fill! (rival-machine-precisions machine) (rival-machine-initial-precision machine))
+        (backward-pass machine))
```

Comment:
Debugging cruft?

Reply:
Idk, I thought I will leave it as a high-level `adjust` logic